iroha/src/config.rs

```diff
@@ -17,33 +18,40 @@ use crate::{
     wsv::config::Configuration as WorldStateViewConfiguration,
 };
 
-const IROHA_PUBLIC_KEY: &str = "IROHA_PUBLIC_KEY";
-const IROHA_PRIVATE_KEY: &str = "IROHA_PRIVATE_KEY";
-
 /// Configuration parameters container.
-#[derive(Clone, Deserialize, Debug)]
+#[derive(Clone, Deserialize, Serialize, Debug, Configurable)]
 #[serde(rename_all = "UPPERCASE")]
+#[config(env_prefix = "IROHA_")]
 pub struct Configuration {
     /// Public key of this peer.
+    #[config(serde_as_str)]
     pub public_key: PublicKey,
     /// Private key of this peer.
     pub private_key: PrivateKey,
```

Comment:
Don't we need `serde_as_str` also for private_key ?

Reply:
Private key is structure actually, so it is represented by json object

Reply:
Ok